ui-kit/HorizontalHighlightCard/HorizontalHighlightCard.js

```diff
@@ -7,6 +7,7 @@ import Styled from 'ui-kit/DefaultCard';
 const HorizontalHighlightCard = (props = {}) => {
   let height = props?.height;
   let mobileHeight = props?.mobileHeight || 250;
+  let iPadHeight = props?.iPadHeight || 350;
```

Comment:
@leoalfonso14 Great thinking here adding extra viewport, can we rename this to `tabletHeight` instead of `iPadHeight` just so the language is more universal?